projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/tools/broad_axe.json

```
@@ -0,0 +1,19 @@
+{
+  "tool": "tconstruct:broad_axe",
+  "text": [
+    {
+      "text": "板斧是一柄效率很高的砍树工具。一斧子就能砍到整棵树。"
```

```suggestion
      "text": "板斧是一种效率很高的砍树工具。一斧子就能砍倒整棵树。"
```
一柄？一柄一般指的是具体某一把斧子吧，这里说的是板斧这种工具。